mobile/lib/components/ra_bottomnavbar.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:radioaktywne/extensions/build_context.dart';
+
+import 'package:radioaktywne/resources/assets.gen.dart';
+import 'package:radioaktywne/resources/colors.dart';
+
+class RaBottomNavigationBar extends StatelessWidget {
+  const RaBottomNavigationBar({
+    super.key,
+    required this.currentIndex,
+  });
+
+  final int currentIndex;
+
+  static final labels = [
+    'home',
+    'mic',
+    'album',
+    'article',
+  ];
+
+  SizedBox _loadIconWithName(String name) {
+    return SizedBox(
+      width: 35,
+      height: 35,
+      child: SvgGenImage('assets/icons/$name.svg').svg(),
+    );
+  }
+
+  List<BottomNavigationBarItem> _makeList(List<String> labels) {
+    final list = <BottomNavigationBarItem>[];
+    for (final label in labels) {
+      list.add(
+        BottomNavigationBarItem(
+          icon: _loadIconWithName('${label}_outline'),
+          activeIcon: _loadIconWithName(label),
+          label: label,
+        ),
+      );
+    }
+    return list;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.only(top: 8),
+      height: 79,
```

Comment:
according to Figma - the teal strip should be 5 pixels tall, and the whole navbar 50 pixels tall.
![obraz](https://github.com/RadioAktywne/ra-app/assets/35864414/acca27a7-1ef4-454f-8e94-cafdcea636bf)
![obraz](https://github.com/RadioAktywne/ra-app/assets/35864414/69a296bb-e087-426b-b947-d2ed51d48bc8)


```suggestion
      padding: const EdgeInsets.only(top: 5),
      height: 50,
```

This will most likely produce errors with not enough space for the icons though - there is a clever workaround for this. In `BottomNavigationBar` set fontSize to 0 with these parameters to free some unused space:
```
        selectedFontSize: 0,
        unselectedFontSize: 0,
```